crypto/http/http_client.c

```diff
@@ -175,9 +198,10 @@ int OSSL_HTTP_REQ_CTX_set_request_line(OSSL_HTTP_REQ_CTX *rctx, int method_POST,
     if (path[0] != '/' && BIO_printf(rctx->mem, "/") <= 0)
         return 0;
 
-    if (BIO_printf(rctx->mem, "%s "HTTP_PREFIX"1.0\r\n", path) <= 0)
+    if (BIO_printf(rctx->mem, "%s "HTTP_PREFIX_VERSION"0\r\n", path) <= 0)
```

Comment:
Better comment on end of line, the real value : `/* HTTP/1.0 */`
As it was done in other places

Reply:
Yeah, I think it's even better to factor out all these uses - done as follows:
```
#define HTTP_PREFIX_V1_0 HTTP_PREFIX_VERSION"0" /* "HTTP/1.0" */
```